src/main/java/step3/view/OutputView.java

```diff
@@ -0,0 +1,63 @@
+package step3.view;
+
+import step3.domain.*;
+
+import java.text.DecimalFormat;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class OutputView {
+
+    private static final String LOTTO_COUNT_MESSAGE = "%d개를 구매했습니다.";
+    private static final String WINNING_MESSAGE = "%d개 일치 (%d원)- %d개";
+    private static final String WINNING_MESSAGE_BONUS = "%d개 일치, 보너스 볼 일치 (%d원)- %d개";
+    private static final String EARNING_RATE_MESSAGE = "총 수익률은 %s입니다.";
+
+    public static void printPurchaseComplete(Lottos lottos){
+        printLottoCount(lottos);
+        printEachLotto(lottos);
+    }
+
+    private static void printLottoCount(Lottos lottos) {
+        System.out.println(String.format(LOTTO_COUNT_MESSAGE, lottos.lottos().size()));
+    }
+
+    private static void printEachLotto(Lottos lottos) {
+        for(Lotto lotto: lottos.lottos()){
+            String lottoNumbers = lotto.lottoNumbers().stream()
+                    .map(LottoNumber::number)
+                    .sorted()
+                    .map(String::valueOf)
+                    .collect(Collectors.joining(", ", "[", "]"));
+            System.out.println(lottoNumbers);
+        }
+    }
+
+
+    public static void print(WinningResult result) {
+        printTitle();
+        printRankInfo(result);
+        printFinalEarningRate(result);
+    }
+
+    private static void printFinalEarningRate(WinningResult result) {
+        DecimalFormat decimalFormat = new DecimalFormat("#.##");
+        System.out.println(String.format(EARNING_RATE_MESSAGE, decimalFormat.format(result.getEarningRate())));
+    }
+
+    private static void printRankInfo(WinningResult result) {
+        for(Map.Entry<LottoRank, Integer> info : result.getResult().entrySet()){
+            LottoRank rank = info.getKey();
+            if(rank == LottoRank.SECOND){
```

Comment:
```suggestion
            if (rank.isSecond()){
```
메시지를 직접 전달하는 것이 좋을 것 같습니다.

Reply:
추가로 for 내부 depth도 줄여보면 좋겠네요

Reply:
반영하겠습니다